packages/db/README.md

```diff
@@ -87,6 +87,13 @@ npm run stop:clean
 
 5. If the schema changes include creating a new table, type or view update `reset.sql` as well.
 
+6. Add the required migration script(s) to [postgres/migration](./postgres/migrations/) folder. Please follow the naming convention and add an incremental number suffix to the name of the migration file.
```

Comment:
Minor typo: should be `postgres/migrations`.
Also, the incremental number should be a prefix?